### PR TITLE
[RTM] Feature image size for provide responsive image

### PR DESCRIPTION
**Pull request by baumannsven**
This pull request is a part of this.

https://github.com/MetaModels/core/pull/1211

### CODE REVIEW

File: contao/dca/tl_metamodel_rendersetting.php [hunk 1]
```diff
@@ -65,13 +66,14 @@
 
 $GLOBALS['TL_DCA']['tl_metamodel_rendersetting']['fields']['file_imageSize'] = array
 (
-    'label'                   => &$GLOBALS['TL_LANG']['tl_metamodel_rendersetting']['file_imageSize'],
+    'label'                   => &$GLOBALS['TL_LANG']['tl_metamodel_rendersetting']['file_imageSizef'],
     'exclude'                 => true,
     'inputType'               => 'imageSize',
-    'options'                 => $GLOBALS['TL_CROP'],
+    'options'                 => \Contao\System::getImageSizes(),
```
Comment (discordier) [inline contao/dca/tl_metamodel_rendersetting.php]: Can we make this an `option_callback` to ease migration of the feature to Contao 4 please (we want a service there)?

Comment (baumannsven) [reply contao/dca/tl_metamodel_rendersetting.php]: Yeah, that's a good idea. Solve this with event.

Comment (zonky2) [reply contao/dca/tl_metamodel_rendersetting.php]: @baumannsven ready?

Comment (baumannsven) [reply contao/dca/tl_metamodel_rendersetting.php]: The image size options comes yet over the GetPropertyOptionsEvent

File: contao/dca/tl_metamodel_rendersetting.php [hunk 2]
```diff
@@ -65,13 +66,14 @@
 
 $GLOBALS['TL_DCA']['tl_metamodel_rendersetting']['fields']['file_imageSize'] = array
 (
-    'label'                   => &$GLOBALS['TL_LANG']['tl_metamodel_rendersetting']['file_imageSize'],
+    'label'                   => &$GLOBALS['TL_LANG']['tl_metamodel_rendersetting']['file_imageSizef'],
```
Comment (richardhj) [inline contao/dca/tl_metamodel_rendersetting.php]: Probably a typo here.

Comment (baumannsven) [reply contao/dca/tl_metamodel_rendersetting.php]: Thanks

Comment (zonky2) [reply contao/dca/tl_metamodel_rendersetting.php]: dachte, wäre die Schwester von "Axelf"
:D

Comment (richardhj) [reply contao/dca/tl_metamodel_rendersetting.php]: Wer ist eigentlich Axelf? :D

**Comment by zonky2** [comment]
@baumannsven pls update PR with https://github.com/MetaModels/core/issues/1215

**Pull request merged**